src/main/java/ahmad/processor/task/TaskList.java

```diff
@@ -0,0 +1,96 @@
+package ahmad.processor.task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+/**
+ * Class to store list of tasks.
+ */
+public class TaskList {
+    private static final List<Task> list = new ArrayList<Task>();
+
+    private static List<String> getStringList(List<Task> list) {
+        if (list.isEmpty()) {
+            return List.of("No items!");
+        }
+        return List.of(IntStream.range(0, list.size()).mapToObj(i -> (i + 1) + ". " + list.get(i))
+                .reduce("", (acc, cur) -> acc + '\n' + cur));
+    }
+
+    /**
+     * Returns the list of tasks in a single string.
+     *
+     * @return A list of single element, which is the required string.

```

Comment:
Can remove the "@return" because the return value is obvious from the method summary!